### PR TITLE
Add intel-oneapi-vtune package 

### PR DESCRIPTION
**Pull request by vdanilin**
- Add intel-oneapi-vtune package to Spack list
It will install [Intel® VTune™ Profiler](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/vtune-profiler.html) in driverless mode. Provided instructions in description
- Remove 'danvev' from the maintainer list as I'm no longer responsible for Intel integration to Spack 

FYI @rscohn2

### CODE REVIEW

**Comment by vdanilin** [comment]
@rscohn2 could you review this PR? Thanks!

**Review by alalazo — COMMENTED** [review]
Waiting for @rscohn2 review. I see that @danvev is not a valid handle, so fine with me to remove it as a maintainer.

**Review by rscohn2 — APPROVED** [review]
This looks good. Thanks for doing the investigation and finding an installation option that works well with spack.

And thanks for your contributions for the releases and packages.

**Review by alalazo — APPROVED** [review]

**Pull request merged**
